### DISCUSSION TITLE
What is the purpose of the lottie-ios dep

### DISCUSSION THREAD

**Discussion by jacksonHendric** [lottie-react-native/lottie-react-native#926] [Q&A] [answered]
Hi,

What is the purpose of installing both lottie-react-native + lottie-ios?

I tried only installing lottie-react-native (expo) and it works fine (not very stable)
Are there any issues created by not having lottie-ios installed?

Thanks!

**Comment by emilioicai** [comment, marked as answer]
That was a dependency legacy, next version of lottie-react-native should be 100% standalone

**Comment by jacksonHendric** [comment]
Got it thanks! How is v5.0.1 working without Lottie-iOS though?

**Comment by dtodereanfourthline** [comment]
Great improvement! When are you planning to release? :) Looking forward to no lottie-ios

Also, shouldn't it be removed from the podfile?
The problem that I am encountering is that I am need it to be dynamic, and the way you integrate it is static and I have conflicts in my project. Thank you
